### ISSUE TITLE
Make Opaque(byte[]) Public

### ISSUE THREAD

**Issue by lextudio-support** [lextudio/sharpsnmplib#333]
In the v8.0 release of the library, the Lextm.SharpSnmpLib.Opaque constructor that takes a byte array as the parameter has internal accessibility.  I think it should be public.  I understand that you consider Opaque to be deprecated, but this limits the capability of the library.  For example, I would like to simulate an SNMP agent that defines Opaque types.

One place you will find usage of the Opaque type is in the [UCD-SNMP-MIB extensions' Textual Conventions](http://net-snmp.sourceforge.net/docs/mibs/ucdavis.html#textconventions).

**Original Reported Date**: 2014-05-12T14:57:41.33-07:00
**Planned For Release**: 
**Original CodePlex ID**: 7262

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

Fixed in vNext. You can build from source code and fix this issue in v8.0.

**Original Posted Date**: 2014-09-06T03:23:44.083-07:00

**Comment by lextudio-support** [comment]
**Original Closed Date**: 2014-09-06T03:23:48.957-07:00
